documents/src/pages/elements/radio-button.md

```diff
@@ -74,24 +74,51 @@ More than one `ef-radio-button` can be grouped by setting the same value to the
 
 `checked-changed` is the **only** event fired by `ef-radio-button`. It is dispatched whenever the state has been changed by user interaction, such as a click, tap or keyboard event.
 
```

Comment:
Let's add another paragraph describing about adding a wrapper element so only one event listener is required. This links with our live example.

Reply:
Updated!